app/views/shared/_date_time_form_part.html.erb

```diff
@@ -18,20 +18,11 @@
       <div class="date-field-wrapper">-</div>
       <div class="date-chooser-values">
         <label class="date-chooser-select form-label js-only" data-initial-date="<%= initial_date_type %>">
-          <span class="today" data-date-display="<%= today.to_s(:full) %>" data-day="<%= today.day %>" data-month="<%= today.month %>" data-year="<%= today.year %>"><%= t('today') %></span>
-          <span class="tomorrow" data-date-display="<%= tomorrow.to_s(:full) %>" data-day="<%= tomorrow.day %>" data-month="<%= tomorrow.month %>" data-year="<%= tomorrow.year %>"><%= t('tomorrow') %></span>
+          <span class="today" data-date-display="<%= today.to_s :full %>" data-day="<%= today.day %>" data-month="<%= today.month %>" data-year="<%= today.year %>"><%= t('today') %></span>
+          <span class="tomorrow" data-date-display="<%= tomorrow.to_s :full %>" data-day="<%= tomorrow.day %>" data-month="<%= tomorrow.month %>" data-year="<%= tomorrow.year %>"><%= t('tomorrow') %></span>
```

Comment:
Javascript no longer uses `data-day`, `data-month`, and `data-year` attributes. Looks like they are not needed. Suggest we remove them.